ddtrace/contrib/internal/fastapi/patch.py

```diff
@@ -74,14 +75,15 @@ def patch():
 
     if not isinstance(fastapi.BackgroundTasks.add_task, ObjectProxy):
         _w("fastapi", "BackgroundTasks.add_task", _trace_background_tasks(fastapi))
-
     # We need to check that Starlette instrumentation hasn't already patched these
     if not isinstance(fastapi.routing.APIRoute.handle, ObjectProxy):
         _w("fastapi.routing", "APIRoute.handle", traced_handler)
 
     if not isinstance(fastapi.routing.Mount.handle, ObjectProxy):
         _w("starlette.routing", "Mount.handle", traced_handler)
 
+    _on_iast_fastapi_patch()
```

Comment:
would it be better to have something like:

```python
if iast_is_enabled():
    from ddtrace.appsec._iast._patch import _on_iast_fastapi_patch
    _on_iast_fastapi_patch()
```

Are we concerned about any side effects from importing `iast` when it isn't going to be used?